### PR TITLE
NK-65 import and export

### PR DESCRIPTION
**Pull request by NinoHelfenstein**
added csv and json import and export

### CODE REVIEW

File: src/controller.js [hunk 1]
```diff
@@ -28,10 +28,10 @@ const controller = (() => {
     const existingNoteTemplate = document.getElementById(
       "existing-note-template"
     );
-    const newNode = existingNoteTemplate.cloneNode(true);
```
Comment (brukri) [inline src/controller.js:31]: Please keep naming as it is a DOM-node

File: src/controller.js [hunk 2]
```diff
@@ -46,18 +46,26 @@ const controller = (() => {
...
-    const elementId = `existing-node-list-${noteId}`;
-    newNode.setAttribute("id", elementId);
-    existingNotes.appendChild(newNode);
-    document.querySelector(`#${elementId} #existing-note-list-title`).innerText =
-      title;
+    const existingNoteTemplate = document.getElementById(
+      "existing-notes-list-template"
+    );
+    const newNote = existingNoteTemplate.cloneNode(true);
```
Comment (brukri) [inline src/controller.js]: Maybe prefer `node` here as well.

File: src/controller.js [hunk 3]
```diff
@@ -195,16 +206,167 @@ const controller = (() => {
...
+    value = document.getElementById("exportFile").value;
+    if (value === "JSON") {
+      exportJSON();
+    } else {
+      exportCSV();
+    }
+  }
+
+  function exportCSV() {
+    var notes = JSON.parse(localStorage.getItem(SIMPLE_NOTES_STORAGE_KEY));
```
Comment (brukri) [inline src/controller.js]: Prefer `const` or `let` instead of `var`

File: src/controller.js [hunk 4]
```diff
@@ -195,16 +206,167 @@ const controller = (() => {
...
+
+    reader.onload = function (event) {
+      var dataURL = event.target.result;
+    };
+
+    reader.readAsDataURL(file);
+    var name = file.name;
+    console.log(name);
+    if (name.search(/\.csv+$/i) !== -1) {
+      console.log("csvFile");
```
Comment (brukri) [inline src/controller.js]: Remove `console.log`

**Review by brukri — CHANGES_REQUESTED** [review]

**Comment by NinoHelfenstein** [comment]
@brukri

**Comment by NinoHelfenstein** [comment]
@brukri all requested changes made

**Review by brukri — APPROVED** [review]

**Pull request merged**